west.yml

```diff
@@ -235,7 +235,7 @@ manifest:
       groups:
         - tee
     - name: trusted-firmware-a
-      revision: d29cddecde614d81cbec1fb0086cdaebd77d3575
+      revision: 021b58cc4a6df475323185b934b96645ce03c201
```

Comment:
This is not the merged commit. Please use 28f5e137837f1c1a7a7b2af2dd8bb778c0a27532

Reply:
Done, apologies for the mistake